core/trino-spi/src/main/java/io/trino/spi/predicate/SortedRangeSet.java

```diff
@@ -842,11 +842,27 @@ public String toString(ConnectorSession session)
 
     private String formatRanges(ConnectorSession session, int limit)
     {
-        return Stream.concat(
-                IntStream.range(0, min(getRangeCount(), limit))
+        if (isNone()) {
```

Comment:
Are there a test cases for these? I don't see any tests with `...`?

Reply:
Tests with `...` are in subsequent commits where the `toString` method with `limit` parameters is actually exposed.